plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/index.tsx

```diff
@@ -322,6 +328,28 @@ const DatasetCard: React.FC<Props> = ({
   drag(dragRef);
   drop(preview(previewRef));
 
+  const downloadData = async (url: string, name?: string) => {
+    const res = await fetch(url, {
+      method: "GET",
+    });
+    const blob = await res.blob();
+    fileDownload(blob, name ?? "export-data");
+  };
+
+  const exportData = async () => {
```

Comment:
useCallback
handleDataExport

Reply:
Done!

Reply:
Sorry can you update the name of the callback too?

Reply:
Done!